msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -32,27 +36,53 @@ To get started, ensure that you have the following tools:
 
 1. Clone the repository to test the sample app.
 
-   ```bash
+   ```
    git clone https://github.com/microsoft/teams-ai.git
    ```
 
-1. Run the following command to go to the app folder:
+1. Go to the **JS** folder.
+
+   ```
+   cd samples/js
+   ```
+
+1. Enter the following code to install dependencies:
+
+   ```
+   yarn install
+   ```
+
+1. Enter the following code to build dependencies:
+
+   ```
+   yarn build
+   ```
+
+1. Go to the **04.ai.a.teamsChefBot** sample folder.
```

Comment:
As of this comment, please look at Teams Chef Bot for the latest instructions on how to set up the sample.

Reply:
https://teams.microsoft.com/l/message/19:3c493705-8971-4e22-9830-967ad65cc74a_fdd46bf8-457e-4486-a314-935acf94ca94@unq.gbl.spaces/1701124917276?context=%7B%22contextType%22%3A%22chat%22%7D

Reply:
Hi @corinagum,

I don't have access to the link https://teams.microsoft.com/l/message/19:3c493705-8971-4e22-9830-967ad65cc74a_fdd46bf8-457e-4486-a314-935acf94ca94@unq.gbl.spaces/1701124917276?context=%7B%22contextType%22%3A%22chat%22%7D.

But I've tested the chefbot for JS and updated the steps by referring https://github.com/microsoft/teams-ai/blob/main/getting-started/OTHER/README.md.

Reply:
Verified the code with @kavin, Confirmed the code is correct.